sdk/tests/conformance2/transform_feedback/transform_feedback.html

```diff
@@ -439,8 +439,12 @@
     verifyTransformFeedbackVarying(program, 1, true, "out_mul");
     verifyTransformFeedbackVarying(program, 2, false);
 
-    // Now relink.
-    gl.linkProgram(program);
+    // Now recompile/relink the program
+    var skipCompileStatus = true;
+    program = wtu.setupTransformFeedbackProgram(gl, ["vshader", "fshader"],
+        ["out_add", "out_mul"], gl.SEPARATE_ATTRIBS,
+        ["in_data"], skipCompileStatus);
```

Comment:
It looks to me like "skipCompileStatus" is matching the "opt_locations" argument of setupTransformFeedbackProgram instead of "opt_skipCompileStatus".